Documentation/Books/AQL/DataQueries.md

```diff
@@ -286,26 +286,38 @@ must be known to the AQL executor at query-compile time and cannot change at
 runtime. Using a bind parameter to specify the
 [collection name](../Manual/Appendix/Glossary.html#collection-name) is allowed.
 
-Data-modification queries are restricted to a single modify operation per collection.
-That means you may not place several `REMOVE` or `UPDATE` statements for one collection 
-in one query. In case you have several places in a query providing you lists of documents
-to delete, collect them in an array so you can remove them all at once.
-
-Only a single data-modification operation can be used per AQL query. Data-modification
-queries cannot be used inside subqueries. Data-modification operations can optionally
-be followed by `LET` operations and a single `RETURN` operation to return data. If
-expressions are used within these operations, they cannot contain subqueries or 
-access data in collections using AQL functions.
-
-Finally, data-modification operations can optionally be followed by `LET` and `RETURN`,
-but not by other statements such as `SORT`, `COLLECT` etc.
+Data-modification operations are restricted to one collection at a time. It is
+not possible to use multiple data-modification operations for the same collection
+in the same query, or follow up a data-modification operation for a specific 
+collection with a read operation for the same collection. Neither is it possible
+to follow up any data-modification operation with a traversal query (which may
+read from arbitrary collections not necessarily known at the start of the
+traversal).
+
+That means you may not place several `REMOVE` or `UPDATE` statements for the same 
+collection into the same query. It is however possible to modify different collections
+by using multiple data-modification operations for different collections in the
+same query.
+In case you have a query with several places that need to remove documents from the
+same collection, it is recommended to these documents in an array and have the 
```

Comment:
it is recommended to < collect > these documents < or their `_key`s >

Reply:
changes done as suggested. please recheck